src/groupeddataframe/grouping.jl

```diff
@@ -338,7 +338,8 @@ function combine(f::Any, gd::GroupedDataFrame)
     end
 end
 combine(gd::GroupedDataFrame, f::Any) = combine(f, gd)
-combine(gd::GroupedDataFrame, f::Pair...) = combine(f, gd)
```

Comment:
That comment still applies.

Reply:
yes. It still works without this. It's deleted

Reply:
It turns out it is needed. `combine(f, (:c => sum, :d => mean))` breaks without it. 

I thought 

```
combine(gd::GroupedDataFrame, f::Union{Pair, AbstractVector{<:Pair}}...) 
```

would capture it, but it does not. If we wanted to avoid the dispatch we could add a method for `combine` which collects a tuple of pairs into a `Vector`.